atoma-proxy/src/server/handlers/nodes.rs

```diff
@@ -322,7 +322,7 @@ pub(crate) async fn nodes_create_lock(
                 endpoint: NODES_CREATE_LOCK_PATH.to_string(),
             })?
             .map_err(|e| AtomaProxyError::InternalError {
-                message: format!("Failed to get GetCheapestNodeForModel result: {:?}", e),
+                message: format!("Failed to get GetCheapestNodeForModel result: {e:?}"),
```

Comment:
Do we need the `:?`?

Reply:
Yes see https://github.com/atoma-network/atoma-proxy/pull/175#discussion_r1936051622